web/src/stream_data.js

```diff
@@ -33,8 +33,8 @@ class BinaryDict {
             - autocomplete stream in compose
     */
 
-    trues = new FoldDict();
-    falses = new FoldDict();
+    trues = new Map();
+    falses = new Map();
```

Comment:
This commit should note that 91d32d9f168b878acf536178a2d40131cb337bc0 is what made this possible, right?

Reply:
Yeah mentioned this commit in the message 👍